libs/spice/Makefile

```diff
@@ -32,30 +34,45 @@ define Package/libspice-server
   DEPENDS:=+glib2 +libjpeg +libopenssl +pixman +zlib
 endef
 
-define Package/libspice-server/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspice-server.so* $(1)/usr/lib
-endef
+# audio codec
+MESON_ARGS += \
+	-Dcelt051=disabled \
+	-Dopus=disabled
+
+MESON_ARGS += \
+	-Dgstreamer=no \
+	-Dlz4=false \
+	-Dsasl=false \
+	-Dsmartcard=disabled \
+	-Dalignment-checks=false \
+	-Dextra-checks=false \
+	-Dstatistics=false \
+	-Dmanual=false \
+	-Dinstrumentation=no \
+	\
```

Comment:
Why this fragile form? Why not just MESON_ARGS += again? (personally, I'd do that for _every_ arg, so you don't have diffs based on the last \ being present or not when sorting/adding new options, but at least, putting in artificial line breaks seems pointless

Reply:
This is to separate the main arguments from the subproject ones.

Reply:
I second the action to make it another `MESON_ARGS +=` for the subproject.

As for the diff generation on presence of last backslash, usually I always keep a backslash at the end.  It's convenient when moving them around, adding or removing some of them.  `utils/qemu/Makefile` as an example.

Reply:
Done.